docs(troubleshooting): refactor some troubleshooting section

benjaminParisel opened the pull request:
Refactor troubleshooting sections to follow the right syntax in some files

* https://bonitasoft-bonita-doc-build_preview-pr-2178.surge.sh/bonita/2021.1/bonita-bpm-platform-setup
* https://bonitasoft-bonita-doc-build_preview-pr-2178.surge.sh/bonita/2021.1/generate-pdf-from-an-office-document
* https://bonitasoft-bonita-doc-build_preview-pr-2178.surge.sh/bonita/2021.1/bonita-bpm-studio-preferences

benjaminParisel committed:
docs: Use new syntax for troubleshooting's section

benjaminParisel committed:
And another page

tbouffard commented:
Nice, this will be a great example to show to people that have to write troubleshooting guide 👍🏿 You rock @benjaminParisel

vbotnari12 committed:
Update bonita-bpm-platform-setup.adoc

changing the wording regarding setup tool. note for the future: tenant related instructions to be adapted

vbotnari12 committed:
Update bonita-bpm-studio-preferences.adoc

display adjustments

vbotnari12 committed:
Update generate-pdf-from-an-office-document.adoc

adapting the display

vbotnari12 committed:
Update bonita-bpm-studio-preferences.adoc

vbotnari12 committed:
Update bonita-bpm-platform-setup.adoc

vbotnari12 committed:
Update bonita-bpm-platform-setup.adoc

vbotnari12 committed:
Update bonita-bpm-studio-preferences.adoc

vbotnari12 committed:
Update generate-pdf-from-an-office-document.adoc

vbotnari12 reviewed (approved):
Looks beautiful to me
